ztag/transforms/dns.py

```diff
@@ -12,38 +13,58 @@ class DNSTransform(ZMapTransform):
     def __init__(self, *args, **kwargs):
         super(DNSTransform, self).__init__(*args, **kwargs)
 
+    def _transform_responses(self, responses):
+        error = False
+        ret = []
+
+        for response in responses:
+            if response["rdata_is_parsed"] == 1:
+                r = dict()
+                r["name"] = response["name"]
+                r["type"] = response["type_str"]
+                r["response"] = response["rdata"]
+                ret.append(r)
+            else:
+                error = True
+
+        return (error, ret)
```

Comment:
You can have an error and data to return. Example, two answers, first is parsed correctly the second is broken.